generation/hermetic_split_repo.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+
+set -ef
+
+# This script takes the following variables
+if [ -z "$REPO" ]; then
+  echo "Please set REPO. E.g., REPO=https://github.com/googleapis/java-spanner"
```

Comment:
nit: do we want to include a link for the git cloning via ssh as well?

Reply:
"https" is just fine.